src/agent/src/rpc.rs

```diff
@@ -378,7 +403,7 @@ impl AgentService {
 
         let p = find_process(&mut sandbox, cid.as_str(), eid.as_str(), init)?;
 
-        let mut signal = Signal::try_from(req.signal as i32).unwrap();
+        let mut signal = Signal::try_from(req.signal as i32)?;
```

Comment:
The error message direct from `try_from` probably won't be very useful, so you probably want to use a `.context()` or similar here.